### DISCUSSION TITLE
Adding an existing project to GitHub

### DISCUSSION THREAD

**Discussion by jeffbarish** [community/community#130659] [Other Feature Feedback, Questions, & Ideas]
I am following the directions [here](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/How-to-push-an-existing-project-to-GitHub) for adding an existing project to GitHub. When I get to this step:

git push -u -f origin master

I get a request for a password:

Password for 'https://gobbledygook@github.com':

Naturally, I do not know the password for gobbledygook. I assume that gobbledygook is supposed to be my GitHub username. I tried deleting my repository and starting over, but I still get the same request.

**Comment by jeffbarish** [comment]
Is there a way to delete my account so that I can start over? As I do not know why I am getting a request for a password for a garbage account @github.com and I do not know the password for the garbage account, I know of no other alternative but to delete my account and start over.

**Comment by anhoangcao** [comment]
1. Create a GitHub Repository:
	Go to https://github.com and log in to your account.
	Click the '+' icon in the top right corner and select 'New repository'.
	Name your repository and provide a description (optional).
	Choose whether the repository should be public or private.
	Do not initialize the repository with a README, .gitignore, or license if your project 	already has these files.
	Click 'Create repository'.

2. Initialize Local Git Repository:
	Open a terminal (or Git Bash on Windows).
	Navigate to your project directory:				
		cd path/to/your/project

3. Initialize the Git repository:
	git init

4. Add Files to the Repository:
	Add all files to the repository:
		git add .
	Check the status to see which files will be committed:
		git status
	
5. Commit Changes:

	Commit the changes with a message:
		git commit -m "Initial commit"

6. Connect to GitHub Repository:
	Copy the URL of your GitHub repository. It will look something like 	https://github.com/yourusername/your-repository.git.

7. Add the remote repository:
	git remote add origin https://github.com/yourusername/your-repository.git

8. Push to GitHub:
	Push the local repository to GitHub:	
		git push -u origin master

**Reply by jeffbarish** [reply]
At the last step, a popup appears demanding a password for https://somegarbage@github.com

**Comment by felipeserrano05** [comment]
you can also use your token instead of password.
example: git remote set-url origin https://your_github_username:your_github_token@github.com/your_githubusername/your_repo

**Reply by jeffbarish** [reply]
Using a token, I was able to get past the `git push -u origin master` step. However, the only files that appear in my repository are LICENSE and README.md.

**Comment by jeffbarish** [comment]
Is there a way to delete my account so that I can start over?

**Comment by Chamal1120** [comment]
Approach can be vary depending on some criteria,

Assuming you are pushing your code first time to github I recommend using the cli instead of gui apps.

01. First make sure you have the latest git and github cli versions installed in your system. Try giving an update command also.

02. Then navigate the project directory and initialize git.
 `git init`

03. login to github using gh cli
 `gh auth login`

04. Create a repo
`gh repo create`

05. Add all files to git as initial commit
`git add .`
`git commit -m "Initial commit"`

06. Connect to the repo
`git remote add origin https://github.com/your-username/my-existing-project.git`

07. Push your project
`git push -u origin main`


If everything goes right you should see your repo in your github account when you visit it using a browser.

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @jeffbarish!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the General category. 

With that being said, here's a Docs article on how to delete your account: [Deleting your personal account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account).

I've gone ahead and moved it for you. Good luck!

**Comment by jeffbarish** [comment]
Thanks for all the suggestions. I was able to solve the problem (without deleting my account and starting over) by specifying an SSH authentication key. I still feel completely at sea, but I did successfully create 4 repositories.
